templates/ingress.yaml

```diff
@@ -68,7 +68,7 @@ spec:
 {{- if and .Values.ingress.enabled (eq .Values.airflow.executor "CeleryExecutor") }}
 ---
 kind: Ingress
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
We should change this to something like https://github.com/astronomer/astronomer/blob/c57c70823b/templates/_apivers.tpl#L17-L23

Reply:
Makes sense with 1.19 being new

Reply:
Yeah. We've run into problems where customers updated their cluster but not their chart. We should be putting in these `useLegacyThing` options anywhere that the old APIs are available https://github.com/astronomer/astronomer/blob/master/templates/_apivers.tpl#L53

Reply:
made that change for Ingress items.